pom.xml

```diff
@@ -10,11 +10,13 @@
 
   <artifactId>accurev</artifactId>
   <packaging>hpi</packaging>
-  <version>0.7.17-SNAPSHOT</version>
+  <version>${revision}${changelist}</version>
   <name>Jenkins AccuRev plugin</name>
   <description>This plugin allows you to use AccuRev as a SCM.</description>
   <url>http://wiki.jenkins-ci.org/display/JENKINS/AccuRev+Plugin</url>
   <properties>
+        <revision>0.7.17</revision>
+        <changelist>-SNAPSHOT</changelist>
```

Comment:
BTW [reindent](https://github.com/jenkinsci/incrementals-tools/blob/bc7488dd22fb2ea8813d82ec3ff5caf8dd762bb8/maven-plugin/src/main/java/io/jenkins/tools/incrementals/maven/IncrementalifyMojo.java#L142)

Reply:
I used `mvn incrementals:incrementalify` to increment that accidentally have misindendet I guess so I blame you 😝

Reply:
Yes, hence the link!

Reply:
I missed the link, sorry 😢